Can't install  Node  & npm

### Info:
- Docker version (` 17.09.0-ce-mac35 (19611)`): 
- Laradock commit (`$ git rev-parse HEAD`): 
- System info (Mac): 
- System info disto/version: 

### Issue:
Can't install node, because not installed npm...
_____

### Expected behavior:
npm install  - successfully completed!
_____

### Reproduce:
1. I create dir test1.loc -
2. Edit myProject\laradock\.env. 
3. Change WORKSPACE_INSTALL_NODE=false to WORKSPACE_INSTALL_NODE=true.
4. docker-compose up -d workspace nginx mysql. 
was OK
5. docker-compose exec workspace bash.
6. composer install.
7. php artisan key:generate.
8. After I went to browser - all ok!
9. Go to dir in my project ( cd test1.loc)
10 .npm install.

....error...
_____

### Relevant Code:

```

gyp verb `which` failed   stack: 'Error: not found: /path/to/executable/python2.7\n    at getNotFoundError (/var/www/test1.loc/node_modules/which/which.js:13:12)\n    at F (/var/www/test1.loc/node_modules/which/which.js:68:19)\n    at E (/var/www/test1.loc/node_modules/which/which.js:80:29)\n    at /var/www/test1.loc/node_modules/which/which.js:89:16\n    at /var/www/test1.loc/node_modules/isexe/index.js:42:5\n    at /var/www/test1.loc/node_modules/isexe/mode.js:8:5\n    at FSReqWrap.oncomplete (fs.js:166:21)',
gyp verb `which` failed   code: 'ENOENT' }
gyp ERR! configure error
gyp ERR! stack Error: Can't find Python executable "/path/to/executable/python2.7", you can set the PYTHON env variable.
gyp ERR! stack     at PythonFinder.failNoPython (/var/www/test1.loc/node_modules/node-gyp/lib/configure.js:483:19)
gyp ERR! stack     at PythonFinder.<anonymous> (/var/www/test1.loc/node_modules/node-gyp/lib/configure.js:397:16)
gyp ERR! stack     at F (/var/www/test1.loc/node_modules/which/which.js:68:16)
gyp ERR! stack     at E (/var/www/test1.loc/node_modules/which/which.js:80:29)
gyp ERR! stack     at /var/www/test1.loc/node_modules/which/which.js:89:16
gyp ERR! stack     at /var/www/test1.loc/node_modules/isexe/index.js:42:5
gyp ERR! stack     at /var/www/test1.loc/node_modules/isexe/mode.js:8:5
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:166:21)
gyp ERR! System Linux 4.9.49-moby
gyp ERR! command "/home/laradock/.nvm/versions/node/v9.1.0/bin/node" "/var/www/test1.loc/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /var/www/test1.loc/node_modules/node-sass
gyp ERR! node -v v9.1.0
gyp ERR! node-gyp -v v3.6.2
gyp ERR! not ok
Build failed with error code: 1
npm WARN EINVAL: invalid argument, scandir '/var/www/test1.loc/node_modules/fsevents/node_modules/dashdash/node_modules'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-sass@4.5.3 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-sass@4.5.3 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2017-11-09T17_26_45_943Z-debug.log
```

--- a/workspace/Dockerfile-71
+++ b/workspace/Dockerfile-71
@@ -54,9 +54,7 @@ ENV PUID ${PUID}
 ENV PGID ${PGID}
 
 RUN groupadd -g ${PGID} laradock && \
-    useradd -u ${PUID} -g laradock -m laradock && \
-    apt-get update -yqq && \
-    apt-get install -y python2.7
+    useradd -u ${PUID} -g laradock -m laradock
 
 #####################################
 # SOAP:
@@ -686,8 +684,7 @@ RUN if [ ${INSTALL_DUSK_DEPS} = true ]; then \
 # Clean up
 USER root
 RUN apt-get clean && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
-    npm config set python /usr/bin/python2.7
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 # Set default work directory
 WORKDIR /var/www

